src/cloud/components/RateLimitAlertContent.tsx

```diff
@@ -93,6 +93,7 @@ export const UpgradeContent: FC<UpgradeProps> = ({
         <UpgradeMessage {...{limitText, link, type}} />
         <CloudUpgradeButton
           className="upgrade-payg--button__rate-alert"
+          hidePromoMessage={true}
```

Comment:
There is already a separate part of this component that contains the promo message.